bundles/org.eclipse.jface/src/org/eclipse/jface/viewers/FocusCellOwnerDrawHighlighter.java

```diff
@@ -23,7 +23,7 @@
 import org.eclipse.swt.graphics.Rectangle;
 import org.eclipse.swt.widgets.Event;
 import org.eclipse.swt.widgets.Listener;
-
+import org.eclipse.jface.util.Util;
```

Comment:
Has Organize Imports been used?  It's odd that there is no blank line after the import.

Reply:
done the changes.